package.json

```diff
@@ -101,6 +101,7 @@
     "formiojs": "~4.13.0",
     "jest": "^29.7.0",
     "jest-environment-jsdom": "^29.7.0",
+    "monaco-editor": "^0.48.0",
```

Comment:
where's this dependency coming from, exactly? Is it a peer-dependency? I was under the impression that we load it from a CDN?

Reply:
I added a possible explanation in this commit's message: https://github.com/open-formulieren/formio-builder/pull/153/commits/5c22f78d32c8a6c89441dab79f676ed34b56a6d0

Reply:
ah yes, that makes sense!